pkg/vpcmodel/commonConnectivity.go

```diff
@@ -6,7 +6,114 @@ SPDX-License-Identifier: Apache-2.0
 
 package vpcmodel
 
-import "github.com/np-guard/models/pkg/connection"
+import (
+	"github.com/np-guard/models/pkg/connection"
+	"github.com/np-guard/models/pkg/netp"
+)
+
+// todo: remove stateful from connection.Set (for both options)
+
+// ExtendedSet connection details
+type ExtendedSet struct {
+	statefulConn    *connection.Set // stateful TCP connection between <src, dst>
+	nonStatefulConn *connection.Set // nonstateful TCP connection between <src, dst>; complementary of statefulConn
+	otherConn       *connection.Set // non TCP connection (for which stateful is non-relevant)
+	conn            *connection.Set // entire connection
+}
+
+// operation on ExtendedSet
+// The operations are performed on the disjoint statefulConn and otherConn and on conn which contains them;
+// nonStatefulConn - the tcp complementary of statefulConn w.r.t. conn -
+// is computed as conn minus (statefulConn union otherConn)
+
+func NoConnsExtendedSet() *ExtendedSet {
+	return &ExtendedSet{
+		statefulConn:    NoConns(),
+		nonStatefulConn: NoConns(),
+		otherConn:       NoConns(),
+		conn:            NoConns(),
+	}
+}
+
+var all = connection.All()
```

Comment:
since its a global variable, rename all to something more specific, or remove it

Reply:
removed